book-library/script.js

```diff
@@ -31,14 +31,23 @@ function submit() {
   if (
     title.value == null ||
     title.value == "" ||
+    author.value == null ||
+    author.value == "" ||
     pages.value == null ||
     pages.value == ""
   ) {
     alert("Please fill all fields!");
     return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
+  } else if (Number(pages.value)<= 0){
+    alert("Page number must be bigger than 0!");
```

Comment:
I like the error message. An edge case you might not have thought of is when someone puts something other than numbers in the field - intresting little side project if you want to explore it but not a big issue